### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by topliceanu**

### CODE REVIEW

File: core/services/bulletprooftxmanager/eth_broadcaster.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ const (
 // somewhere has received the transaction successfully.
 //
 // This does not guarantee delivery! A whole host of other things can
-// subsequently go wrong such as transctions being evicted from the mempool,
+// subsequently go wrong such as transitions being evicted from the mempool,
```
Comment (coventry) [inline core/services/bulletprooftxmanager/eth_broadcaster.go:34]: transactions

File: docs/core/BULLETPROOF_TX_MANAGER_ARCHITECTURE.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Conceptually, **EthBroadcaster** assigns a nonce to a transaction and ensures th
 
 It does not guarantee eventual confirmation!
 
-A whole host of other things can subsequently go wrong such as transctions being evicted from the mempool, eth nodes crashing, netsplits between eth nodes, chain re-orgs etc. Responsibility for ensuring eventual inclusion into the longest chain falls on the shoulders of **EthConfirmer**.
+A whole host of other things can subsequently go wrong such as transitions being evicted from the mempool, eth nodes crashing, netsplits between eth nodes, chain re-orgs etc. Responsibility for ensuring eventual inclusion into the longest chain falls on the shoulders of **EthConfirmer**.
```
Comment (coventry) [inline docs/core/BULLETPROOF_TX_MANAGER_ARCHITECTURE.md:96]: transactions

**Comment by topliceanu** [comment]
@coventry Thanks for the review. Good snipes! I updated the PR.

File: belt/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 A toolbelt for performing various commands on chainlink smart contracts.
 This cli tool is currently used within `@chainlink/contracts` for the usage of running
-build and development tools across multiple solidity contract verisions.
+build and development tools across multiple solidity contract revisions.
```
Comment (coventry) [inline belt/README.md:5]: Sorry, found another one: versions

**Pull request merged**